src/app/component/header/header.component.html

```diff
@@ -44,7 +44,7 @@ <h1>Jyoti Technosoft LLP</h1>
                All Rights Reserved
             </div>
             <div class="credits">
-              Designed by <a href="https://jyotitechnosoft.com/" target="_blank">Jyoti Technosoft LLP</a>
+              Designed by <a href="https://jyotitechnosoft.com/"  rel="noopener noreferrer" target="_blank">Jyoti Technosoft LLP</a>
```

Comment:
Instead of giving herf link directly could you please make one separate  route that will use in another pages too

Reply:
Router link is only work for internal routing inside application. Here I want to navigate to another sites So for that case I have to create a new function if I remove href tag from there

Reply:
There is already a special route for this please use that